tests/test_regression.py

```diff
@@ -7,14 +7,66 @@
 from autocorpus.configs.default_config import DefaultConfig
 
 
-def test_autocorpus(data_path: Path):
-    """A regression test for the main Auto-CORPus class on the Auto-CORPus paper.
+def test_pmc_autocorpus():
+    """A regression test for the main autoCORPus class, using the current PMC config on the AutoCORPus Paper.
+
+    The test data output is built using `run_app.py` with the following arguments:
+    ```
+    auto-corpus -b "PMC" -t "tests/data/PMC/Current/" -f "tests/data/PMC/Current/PMC8885717.html"
+    ```
+    """
+    from autocorpus.Autocorpus import Autocorpus
+
+    with open(
+        Path(__file__).parent
+        / "data"
+        / "PMC"
+        / "Current"
+        / "PMC8885717_abbreviations.json",
+        encoding="utf-8",
+    ) as f:
```

Comment:
These should use the `data_path` fixture like the other test. Same should be applies to all the below file paths.
```suggestion
    pmc_example_path = data_path / "PMC" / "Current"
    with open(
        pmc_example_path / "PMC8885717_abbreviations.json",
    ) as f:
```